applications/dashboard/controllers/api/AbstractApiController.php

```diff
@@ -76,11 +79,28 @@ public function getUserFragmentSchema() {
                 'name:s' => 'The username of the user.',
                 'photoUrl:s' => 'The URL of the user\'s avatar picture.',
                 'dateLastActive:dt|n' => 'Time the user was last active.',
+                "permissions:a?" => $this->getPermissionFragment(),
             ], 'UserFragment');
         }
         return $this->userFragmentSchema;
     }
 
+    /**
+     * Get a fragment representing a single permission row.
+     *
+     * @return Schema
+     */
+    protected function getPermissionFragment() {
+        if ($this->permissionsFragment === null) {
+            $this->permissionsFragment = $this->schema([
+                "id:i?",
```

Comment:
Can we add descriptions for these? The "permissions" one in particular seems relevant.

Reply:
Unfortunately not. There seems to be an issue with garden-schema, where expanding on this schema beyond its current state results in erroneous validation errors. I believe we'll need to file an issue against that endpoint, but additional discovery is required.

Reply:
Ah that's what you were talking about! Just update this with that issue once it's created.